core/templates/dev/head/components/summary_tile/ExplorationSummaryTileDirective.js

```diff
@@ -132,6 +133,12 @@ oppia.directive('explorationSummaryTile', [
               if ($scope.getCollectionId()) {
                 result += ('?collection_id=' + $scope.getCollectionId());
               }
+              if ($scope.parentExplorationIds()) {
+                var parameterList = $scope.parentExplorationIds();
+                if (parameterList.length > 1) {
+                  result += UrlService.updateParameterList(parameterList);
```

Comment:
This should be

```
result = UrlService.addParams(result, {
  parent: parentExplorationIds
});
```

Basically, keep all the URL manipulation within URL service; keep all the other context outside.

Reply:
Done